tests/jquery.flot.hover.Test.js

```diff
@@ -0,0 +1,72 @@
+describe("flot hover plugin", function () {
```

Comment:
I think this is a good start for a fist pass. Next PR I expect to see more corner cases :-P